keyboards/ortho5by12/ortho5by12.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 Yiancar
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
This needs to be reformatted to properly match the layout used in the keymap. 

Since I don't know how it's wired up, I can't really help here. 

That said, you can look at the preonic rev3 to see how it maps it. 

Also:
```suggestion
#define LAYOUT_ortho_5x12( \
```

Reply:
the pcb is built and tested as working correctly. although it seems out of order, that's correct for a duplexed switch matrix as was used in this build.

Reply:
I think @drashna's contention isn't the wiring itself (he's just stating he can't verify the guts of the LAYOUT macro directly), but rather naming the LAYOUT macro "LAYOUT" (or, previously "KEYMAP" as it was) and then redefining it as something else.

If I recall from last night, the LAYOUT macro in ortho5by12.h was previously called "KEYMAP", then that was defined as both "LAYOUT" and "LAYOUT_preonic", and referred to in the keymap as "LAYOUT_preonic".

Drashna's suggestion merely proposes calling the LAYOUT "LAYOUT_ortho_5x12" with no further defines and using only that name in the keymap.